src/v2/view-builder/utils/LinksUtil.js

```diff
@@ -109,7 +109,7 @@ const getSignOutLink = (settings) => {
     return [
       {
         'actionPath': 'cancel',
-        'label': loc('signout', 'login'),
+        'label': loc('backToSignin', 'login'),
```

Comment:
Have we thought about `Return to Sign In` instead? That would be inline with `Return to Authenticator List`..

Reply:
@vincentvoong-okta any thoughts on Ani's suggestion?

Reply:
@haishengwu-okta @anipendakur-okta 
This is a good catch. I don’t have a strong preference but IMO “Back to sign in” and “`Back to authenticators list`” is easier to understand than using “Return”.

Separately, that led me to think about how we got to “Return to sign in” and in the first version of OIE designs we used “Sign in with something else” - now that I think about it is probably easier for users to understand and is more human. “Back/return to authenticator list” sounds more like Okta speak. We changed this probably because using “sign in” is not always accurate like for SSPR use cases. An alternative to that might be instead saying “`Use something else`”.